tests/functional/conftest.py

```diff
@@ -135,3 +140,24 @@ def exception_catcher(when: str, outcome):
             else exception_info.exconly()
         exception_logger.error('Unhandled Exception during {}: \n{}'
                                .format(when.capitalize(), str(exc_repr)))
+
+
+
```

Comment:
Excessive emptylinses

Reply:
Minor issue will be fixed in next pending PR:
https://github.com/openvinotoolkit/model_server/pull/812